### DISCUSSION TITLE
latest custom order table version?

### DISCUSSION THREAD

**Discussion by kevinjavitz** [woocommerce/woocommerce#36145] [General]
Hi,

In May 2022 you announced this: https://developer.woocommerce.com/2022/05/16/call-for-early-testing-custom-order-table-migrations/

Is there a more recent update with higher stability that we can download? If so what is the tag on github?

**Comment by vedanshujain** [comment]
hey, it's available from 7.1.0 but currently behind a feature flag. you will need to go to WooCommerce > Settings > Advanced > Features to activate it.
